dvc/repo/pull.py

```diff
@@ -20,11 +25,16 @@ def pull(
         jobs,
         remote=remote,
         all_branches=all_branches,
-        all_tags=all_tags,
         with_deps=with_deps,
+        all_tags=all_tags,
         recursive=recursive,
     )
     self._checkout(
-        targets=targets, with_deps=with_deps, force=force, recursive=recursive
+        targets=targets,
+        with_deps=with_deps,
+        force=force,
+        recursive=recursive,
+
     )
+    logger.info("Data retrieved successfully from DVC remote storage.")
```

Comment:
qq - can we analyze results of self._fetch (processed_files_count in this case) and self._checkout to show a different message if nothing has changed or there are no files to process?

Reply:
Yes, sure. Sorry I was delayed in writing up the message below to say I will start adding in the stats information tomorrow, as don't have time tonight but wanted to address the first set of comments to being with. Thanks again for your feedback.